content/donate/infrastructure.md

```diff
@@ -0,0 +1,54 @@
+# Infrastructure provided by Haiku, Inc.
+As an open-source project, Haiku®'s visibility and presence on the internet is vital to its existence, growth, and reputation. Ensuring high quality services to the Haiku Project is the most essential item for receiving funds.
+
+Some weeks before the release of Haiku R1/Alpha 1, a commitment was made to overhaul the infrastructure that provides key services to the Project. Leading up to this, there had been numerous service issues with the previous shared hosting plan and free source code repository. These issues ranged from unexplainable slowdowns to even occasional downtime. In a few words, the needs of the Project were outgrowing its shoes and an upgrade was desired.
+
+## Dedicated Server
+Through discussions on the [haiku-development mailing list](https://www.freelists.org/list/haiku-development "haiku-development mailing list"), it was decided to acquire an [EX4 dedicated server](https://web.archive.org/web/20161128085411/http://www.hetzner.de/en/hosting/produkte_rootserver/ex4/ "EX4 dedicated server") with Hetzner. This increased the hosting costs from roughly $300/year to $900/year; however it has been worth every cent.
```

Comment:
This info is out of date, the hosting has since been moved elsewhere

Reply:
I've updated that, thanks!